File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    longestConsecutive(nums) {
        let numSet = new Set(nums);

        let max = 0;
        for(let n of numSet){
            if(numSet.has(n - 1)){
                continue;
            }

            let count = 0;
            while(numSet.has(n)){
                count++;
                n++;
            }

            max = Math.max(count, max);
        }

        return max;
    }
}

File: Data Structures & Algorithms/counting-bits/submission-0.py
class Solution:
    def countBits(self, n: int) -> List[int]:
        out = []

        for i in range(n + 1):
            count = 0

            for j in range(32):
                if (1 << j) & i:
                    count += 1

            out.append(count)

        return out